applications/create-substrate-app.md

```diff
@@ -0,0 +1,117 @@
+# create-substrate-app
+
+- **Team Name:** Philoniare Team
+- **Payment Address:** 14AZk92CKjovm4tDvzE9XsjGz3HNeeV2tmnbYitAqyweWoRp (DOT) 
```

Comment:
Please select one of the available payment options (USDC or USDT). That being said, are you aware that 30% of each milestone is paid out in DOT?

Reply:
Updated, that's totally fine. I'd be fine with 100% DOT as well.